packages/vite-plugin-cloudflare/src/plugin-config.ts

```diff
@@ -24,11 +24,16 @@ export interface PluginConfig extends Partial<PluginWorkerConfig> {
 type Defined<T> = Exclude<T, undefined>;
 
 export type AssetsOnlyConfig = SanitizedWorkerConfig & {
+	topLevelName: Defined<SanitizedWorkerConfig['topLevelName']>;
+	name: Defined<SanitizedWorkerConfig['name']>;
+	compatibility_date: Defined<SanitizedWorkerConfig['compatibility_date']>;
```

Comment:
I wonder if `topLevelName`, `name` and `compatibility_date` should be in their own shared base type

Reply:
[793f14b](https://github.com/flarelabs-net/vite-plugin-cloudflare/pull/127/commits/793f14b322e8e534cd201ab7694e4a3946cdd5d5)

I've also updated the types for the plugin config in this file to match those in the docs PR as that won't be merged before we migrate to workers-sdk.